src/Umbraco.Core/Persistence/SqlSyntax/SqlCeSyntaxProvider.cs

```diff
@@ -175,7 +185,7 @@ public override string DeleteDefaultConstraint
         {
             get
             {
-                return "ALTER TABLE [{0}] ALTER COLUMN [{1}] DROP DEFAULT";
+                return "ALTER TABLE {0} ALTER COLUMN {1} DROP DEFAULT";
```

Comment:
Is this not a case if tables have spaces in the names? Or are this only used for umbraco core and not any package?

Reply:
the base `SqlSyntaxProviderBase` already defines a number of these Sql strings, and never escapes the names (eg it does `"ALTER TABLE {0} DROP CONSTRAINT {1}"`) because the names are supposed to be escaped by the code that uses these strings - escaping can vary depending on the database engine (MS Sql vs MySql...).

that one was actually broken and would not work, because of this. so this is aligning with the rest of our code really.